spec/models/curator/controlled_terms/resource_type_spec.rb

```diff
@@ -4,13 +4,25 @@
 require_relative '../shared/controlled_terms/nomenclature'
 require_relative '../shared/controlled_terms/authority_delegation'
 require_relative '../shared/controlled_terms/canonicable'
+require_relative '../shared/controlled_terms/id_from_auth_unique_validatable'
 require_relative '../shared/controlled_terms/reindex_descriptable'
 require_relative '../shared/mappings/mapped_terms'
 
 RSpec.describe Curator::ControlledTerms::ResourceType, type: :model do
   it_behaves_like 'nomenclature'
   it_behaves_like 'authority_delegation'
 
+  it_behaves_like 'id_from_auth_uniqueness_validatable' do
+    # rubocop:disable RSpec/LetSetup
+    let!(:authority) { find_authority_by_code('resourceTypes') }
+    let!(:term_data) { { id_from_auth: 'foo', label: 'Foo' } } # NOTE: Had to use non standard term data here in order for spec to pass
```

Comment:
I think I get what's happening here, we might want to change this comment to be `using invalid data to test that validation fails as expected`. Then standardize this language across all the similar specs.